product_docs/docs/pem/9/considerations/pem_pgbouncer/pem_server_pem_agent_connection_management_mechanism.mdx

```diff
@@ -8,17 +8,23 @@ redirects:
   - /pem/latest/pem_online_help/09_toc_pem_configure_pgbouncer/01_pem_pgbouncer_server_agent_connection/
 ---
 
-Each PEM agent connects to the PEM database server using the SSL certificates for each user. For example, an agent with `ID#1` connects to the PEM database server using the agent1 user.
+## Without PgBouncer
 
-![Connecting to the PEM database without pgBouncer](../../images/pem_database_without_pgbouncer.png)
+In the default configuration, each PEM agent connects to the PEM database server directly using SSL for encryption. Each PEM agent uses its own dedicated user for the connection. 
 
-Prior to PEM version 7.5, the following limitations disallowed the use of the connection pooler between the PEM server and PEM agent:
+For example, a user agent with `ID#1` connects to the PEM database server as agent1.
 
--   The PEM agent uses an SSL certificate to connect to the PEM database server.
--   It uses an individual user identifier when connecting to the PEM database server.
+![Connecting to the PEM database without PgBouncer](../../images/pem_db_without_pgbouncer.png)
 
-EDB modified the PEM agent to allow the agent to use a common database user (instead of the dedicated agent users) to connect to the PEM database server.
+## With PgBouncer
 
-![Connecting to pgBouncer.](../../images/pem_database_with_pgbouncer.png)
+In a PgBouncer-enabled environment, PEM agents cannot connect to the PEM database server directly. PEM agents must use a proxy user that you configure specifically for the connection to PgBouncer. In the example, the proxy user handling all PEM agent connections is called `pem_agent_user1`.
+
+Once the PEM agents connect to PgBouncer using SSL, PgBouncer is responsible for managing connection requests to the PEM database server. PgBouncer uses the rules you have configured for connection pooling to manage the incoming connection requests, for example, by respecting the established maximum number of active connections.
+
+![Connecting to pgBouncer.](../../images/pem_db_with_pgbouncer.png)
+
+!!!note 
+   We recommend using PgBouncer version 1.9.0 or later as the connection pooler. Versions 1.9.0 or later support cert authentication.
```

Comment:
"PgBouncer 1.9.0 or later must be used as the connection pooler."

Let's make it a hard requirement - there's no reason for us to suggest we'll support older versions.